.pre-commit-config.yaml

```diff
@@ -29,9 +30,7 @@ repos:
     rev: ''
     hooks:
       - id: lintr
-        exclude: 'esmvaltool/diag_scripts/extreme_events/climdex.pcic.ncdf'
       - id: style-files  # styler
```

Comment:
When working with this, I noticed that it is much nicer if `styler` comes before `lintr`, because now I get lots of `lintr` warnings about things that `styler` subsequently fixes automatically.